lib/WebpackOptionsDefaulter.js

```diff
@@ -231,6 +231,7 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			return isProductionLikeMode(options) ? 5 : Infinity;
 		});
 		this.set("optimization.splitChunks.automaticNameDelimiter", "~");
+		this.set("optimization.splitChunks.automaticNameMaxLength", 100);
```

Comment:
```suggestion
		this.set("optimization.splitChunks.automaticNameMaxLength", 109);
```

delimiter and hash is included in the maxLength now, so default it to 109 to avoid behavior change.